src/WingetCreateCore/Models/DefaultLocaleManifestModels.cs

```diff
@@ -181,5 +224,7 @@ public partial class DefaultLocaleManifest
             get { return _additionalProperties; }
             set { _additionalProperties = value; }
         }
+    
+    

```

Comment:
Did the generated code add these 2 lines?

Reply:
Yes, 2 extra lines were autogenerated